tests/conftest.py

```diff
@@ -78,7 +78,7 @@ def reset(self):
                 os.unlink(file_path)
             elif os.path.isdir(file_path):
                 shutil.rmtree(file_path)
-        #os.kill(60328, SIGUSR1)
+        #os.kill(76617, SIGUSR1)
```

Comment:
What is this?

Reply:
ignore, it's commented out.  it's support for being able to run the server not as a subprocess for some of my debugging.